x-pack/plugins/security_solution/cypress/e2e/hosts/events_viewer.cy.ts

```diff
@@ -155,4 +160,24 @@ describe('Events Viewer', () => {
         });
     });
   });
+
+  context('Bulk operations', () => {
+    before(() => {
+      esArchiverLoad('bulk_process');
+    });
+    beforeEach(() => {
+      visit(HOSTS_URL);
+      openEvents();
+      waitsForEventsToBeLoaded();
+    });
+
+    it('Adding multiple events to the timeline should be successful', () => {
+      // select all visible events
+      investigateFirstPageEventsInTimeline();
+    });
+
+    it('When selected all events are selected, bulk action should be disabled', () => {
+      investigateAllEventsInTimeline();
```

Comment:
Assertions on tests should not be wrapped inside the tasks methods, should be explicitly written in the test.

Reply:
@MadameSheema .. Sure, I have now removed the assertions from tasks. Please check new updates.